app/ui/src/app/dashboard/dashboard_connections/dashboard-connections.component.html

```diff
@@ -45,14 +45,14 @@ <h1>Create a Connection</h1>
               </div>
 
               <!-- Card Name -->
-              <h2 class="card-pf-title text-center"
+              <h2 class="card-pf-title text-center syn-truncate__ellipsis"
                   title="{{ connection.name }}">
-                {{ connection.name | truncate : '20' : truncateTrail }}
+                {{ connection.name }}
               </h2>
 
               <!-- Card Description / Item Overview -->
-              <p class="card-pf-info text-center">
-                {{ connection.description | truncate : '100' : truncateTrail }}
+              <p class="card-pf-info text-center" title="{{ connection.description }}">
```

Comment:
Although `title="{{ connection.description }}"` is good enough, the more correct approach for attribute data binding in DOM's native attributes is `[attr.title]="connection.description"`. This is just a suggestion for the future, not a change request. ;)